WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/AddCustomAmountPercentageViewModel.swift

```diff
@@ -0,0 +1,51 @@
+import Combine
+import WooFoundation
+import SwiftUI
+import Yosemite
+
+final class AddCustomAmountPercentageViewModel: ObservableObject {
+    let currencyFormatter: CurrencyFormatter
+    let baseAmountForPercentage: Decimal
```

Comment:
I think these can be marked private:

```suggestion
    private let currencyFormatter: CurrencyFormatter
    private let baseAmountForPercentage: Decimal
```

Reply:
Done in 8d1f2b15e723fcee572d81a3d5f0908f55319cfc